<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="2" hostname="DESKTOP-F61H9BC" failures="1" tests="2" name="Default test(failed)" time="0.085" errors="0" timestamp="2022-11-20T15:14:58 IST">
  <testcase classname="testNG_XML.FailedTestCases" name="D" time="0.014">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at testNG_XML.FailedTestCases.D(FailedTestCases.java:20)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- D -->
  <testcase classname="testNG_XML.FailedTestCases" name="A" time="0.003"/>
  <testcase classname="testNG_XML.FailedTestCases" name="E" time="0.0">
    <ignored/>
  </testcase> <!-- E -->
  <testcase classname="testNG_XML.FailedTestCases" name="B" time="0.0">
    <ignored/>
  </testcase> <!-- B -->
</testsuite> <!-- Default test(failed) -->
